CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- [#72] Remove the service environment variables from dogu pods with `enableServiceLinks: false` in the podspec of
+  the dogu pods. `enableServiceLinks: true` would override environment variables from dogus dockerfiles e.g. easyredmine.
```

Comment:
```suggestion
  the dogu pods. Cluster-aware dogus are generally discouraged to use service link env vars because of security considerations. Instead the service DNS names should be used to address these services as described in the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/service/#dns).
```